app/views/layouts/_content.html.haml

```diff
@@ -63,6 +63,9 @@
             = render :partial => "layouts/listnav"
           = yield :left
         #custom_left_cell
+- elsif @angular_app_name
+  .container-fluid{:style => "overflow: hidden; height: 100%;", 'data-ng-app' => "#{@angular_app_name}"}
+    %div{'ui-view' => ''}
 - else
```

Comment:
I guess it's really a question of how we're thinking about the use of router here - is it a prototypal form of the future whole-angular approach, or is it meant to just handle this particular place and nothing else?

If it's the first, then `ng-app` should be on `body` and `ui-view` indeed makes sense in `_content` (even though it should eventually be `ui-view="right_cell"`, so we can have another `ui-view` for explorer trees, listnavs, etc.

If it's the second, then `ui-view` should be in a (ruby)controller-specific partial, and.. maybe so should the `ng-app`, since it doesn't really need to touch anything but the right side, right?